cdserver/features/shelters/step_definitions/shelter_steps.rb

```diff
@@ -8,9 +8,11 @@
                                                   shelter: shelter)
     floor = Floor.find_or_create_by!(shelter_building: building,
                                      name: entry['Floor'])
-    unit = Unit.create!(name: entry['Unit'], floor: floor, shelter: shelter)
-    entry['Beds'].to_i.times do
-      Bed.create!(compartment: unit)
+    if entry['Shelter Type'] == 'Single Adult'
```

Comment:
This is where we need a Scheme.   Part of what a scheme does is lookup an identifer by name.
You ask the ShelterTypeScheme for an identifier for 'Single Adult'  and it gives you back the identifier.